.github/workflows/test-runner-on-perf.yml

```diff
@@ -59,13 +59,13 @@ jobs:
           # write vault tokens
           base64 --decode <<< ${B64_APPLICATION_CREDENTIALS} > ${GOOGLE_APPLICATION_CREDENTIALS}
 
+          gcloud config set compute/zone ${GOOGLE_ZONE}
+          gcloud config set project ${GOOGLE_CLOUD_PROJECT}
+
           gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}
 
-          # configure integration prerequisites
-          gcloud config set compute/zone ${GOOGLE_ZONE} --quiet
-          gcloud config set project ${GOOGLE_CLOUD_PROJECT} --quiet
-          gcloud auth configure-docker --quiet
-          #echo 'Set google sdk to SA user'
+          gcloud auth configure-docker
```

Comment:
Double-checking you want to remove `--quiet`?

Reply:
Thanks for the reminder - I'm going to add it back after the tests run again. 
Perf is finally back up, so it's running now!

Reply:
"--quiet" should be added back in.